src/yuzu/main.cpp

```diff
@@ -4845,6 +4855,27 @@ bool GMainWindow::CheckFirmwarePresence() {
     return true;
 }
 
+void GMainWindow::SetFirmwareVersion() {
+    Service::Set::FirmwareVersionFormat firmware_data{};
+    const auto result = Service::Set::GetFirmwareVersionImpl(
+        firmware_data, *system, Service::Set::GetFirmwareVersionType::Version2);
+
+    firmware_label->setVisible(true);
+
+    if (result.IsError()) {
+        LOG_INFO(Frontend, "Installed Firmware: No firmware available");
+        firmware_label->setText(tr("No firmware"));
+    }
+
+    const std::string display_version(firmware_data.display_version.data());
+    const std::string display_title(firmware_data.display_title.data());
+
+    LOG_INFO(Frontend, "Installed Firmware: {}", display_title);
+
+    firmware_label->setText(tr("Firmware %1").arg(QString::fromStdString(display_version)));
```

Comment:
Seems unnecessary to take up more space in the status bar for the word "Firmware"
```suggestion
    firmware_label->setText(QString::fromStdString(display_version));
```